tests/test-with-none-values.csv

```diff
@@ -1,3 +1,3 @@
-string,bool,bool_with_none,int_with_nan,float_with_nan
```

Comment:
You can call it date with missing or none or null if you like, but definitely not nan...